### PR TITLE
[vsphere] Upgrade ECS to 8.0.0

### PR DESCRIPTION
**Pull request by r00tu53r**
Automated PR.

Upgrades ECS to 8.0.0.

### CODE REVIEW

**Comment by jsoriano** [comment]
Is this change needed? (asked in the meta issue too https://github.com/elastic/integrations/issues/2363#issuecomment-1011956182)

File: packages/vsphere/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "0.1.1"
```
Comment (marc-gr) [inline packages/vsphere/changelog.yml:1]: based on others, should this be a minor bump instead?

Comment (r00tu53r) [reply packages/vsphere/changelog.yml]: fixed.

**Comment by kaiyan-sheng** [comment]
Closing this PR for now. Will reopen once we have time to test migration for this change.

**Pull request closed without merging**